docs/MarkdownInput.md

```diff
@@ -30,4 +29,192 @@ const PostEdit = () => (
+| `helperText`         | Optional | `string`  |            | The helper text to display under the input                                                          |
+| `initialEditType`    | Optional | `string`  | `wysiwyg`  | Markdown editor's initial edit type. Can be `markdown` or `wysiwyg`                                 |
+| `label`              | Optional | `string`  |            | The label. If not set, the label is inferred from the child component                               |
+| `previewStyle`       | Optional | `string`  | `vertical` | Markdown editor's preview style. Can be `tab` or `vertical`                                         |
+| `toolbarItems`       | Optional | `array`   |            | The toolbar items to display in the editor. See [Adding Buttons](#adding-buttons) for more details. |
+| `useCommandShortcut` | Optional | `boolean` | `true`     | Whether use keyboard shortcuts to perform commands                                                  |
+
+`<MarkdownInput>` also accepts the [common input props](https://marmelab.com/react-admin/Inputs.html#common-input-props) and the [editor props](https://nhn.github.io/tui.editor/latest/ToastUIEditorCore) from the [Toast UI](https://ui.toast.com/) editor.
+
+## `source`
+
+Specifies the field of the record that the input should edit. It is required.
+
+{% raw %}
+```jsx
+<Form record={{ id: 123, title: 'Hello, world!', body: '**Lorem Ipsum**' }}>
+    <MarkdownInput source="body" />
+    {/* default value is "**Lorem Ipsum**" */}
+</Form>
+```
+{% endraw %}
+
+## `fullWidth`
+
+You can make the markdown editor full width by setting the `fullWidth` prop to `true`:
+
+{% raw %}
+```jsx
+<SimpleForm>
+    <MarkdownInput source="description" fullwidth />
+</SimpleForm>
+```
+{% endraw %}
+
+## `height`
+
+The editor has a size that can be customized by setting the `height` prop. It is set to `512px` by default. You can use `px` or `%` units.
+
+{% raw %}
+```jsx
+<SimpleForm>
+    <MarkdownInput source="description" height="300px" />
+</SimpleForm>
+```
+{% endraw %}
+
+## `helperText`
+
+If you need to display a text below the markdown editor (usually to explain the expected data to the user), use the `helperText` prop.
+
+{% raw %}
+```jsx
+<SimpleForm>
+    <MarkdownInput
+        source="description"
+        helperText="Enter a description of the post"
+    />
+</SimpleForm>
+```
+{% endraw %}
+
+## `initialEditType`
+
+This prop allows to set the initial edit type of the editor. It accepts `markdown` or `wysiwyg` and is set to `wysiwyg` by default.
+
+{% raw %}
+```jsx
+<SimpleForm>
+    <MarkdownInput source="description" initialEditType="markdown" />
+</SimpleForm>
+```
+{% endraw %}
+
+## `label`
+
+You can customize the label by setting the `label` prop. It is inferred from the `source` prop by default.
+
+{% raw %}
+```jsx
+<SimpleForm>
+    <MarkdownInput source="description" label="Description" />
```

Comment:
bad example: in this case, you don't need to set the label as it's automatically inferred.